cmd/kubeadm/app/util/version_test.go

```diff
@@ -429,12 +431,12 @@ func TestValidateStableVersion(t *testing.T) {
 		{
 			name:          "invalid: error parsing the remote version",
 			remoteVersion: "invalid-version",
-			clientVersion: "v1.12.0",
+			clientVersion: constants.MinimumKubernetesVersion,
 			expectedError: true,
 		},
 		{
 			name:          "invalid: error parsing the client version",
-			remoteVersion: "v1.12.0",
+			remoteVersion: constants.MinimumKubernetesVersion,
```

Comment:
`MinimumKubernetesVersion` is not needed to test the logic in `validateStableVersion`